.buildkite/pipelines/artifacts.yml

```diff
@@ -24,3 +24,9 @@ steps:
     agents:
       queue: n2-4-virt
     timeout_in_minutes: 20
+
+  - command: .buildkite/scripts/steps/artifacts/docker_context.sh
+    label: 'Docker Build Context'
+    agents:
+      queue: n2-4
```

Comment:
n2-standard-4 seems like overkill to build the container, right? Maybe `n2-2` would be more appropriate.

Reply:
https://github.com/elastic/kibana/pull/129399/commits/47cc07d0d152fdf684d44f76633edfb5ebb58c9c